scripts/rebuild.sh

```diff
@@ -0,0 +1,12 @@
+#/bin/bash
+
+cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
+if [ ! -f ../vcpkg ]
+then
+    vcpkg_commit=8dddc6c899ce6fdbeab38b525a31e7f23cb2d5bb
```

Comment:
might wanna consider making this an argument to the bash script; but also we need to figure out how to run this on a regular basis.

Reply:
Yep, that's part of a followup PR